### PR TITLE
Re-build libmo-unpack et al.

### PR DESCRIPTION
**Pull request by pelson**
Works around https://github.com/SciTools/libmo_unpack/issues/18

### CODE REVIEW

**Comment by pelson** [comment]
This should fix

```
$ python -c "import iris"
...
ImportError: /lib64/libc.so.6: version `GLIBC_2.14' not found (required by /opt/conda/lib/python2.7/site-packages/cartopy/../../../libproj.so.9)
```

**Comment by pelson** [comment]
Also need to remember:

```
grib_dump --help
grib_dump: /lib64/libc.so.6: version `GLIBC_2.14' not found (required by /opt/conda/bin/../lib/libgrib_api.so.0)
grib_dump: /lib64/libc.so.6: version `GLIBC_2.14' not found (required by /opt/conda/bin/../lib/./libjasper.so.1)
```

**Comment by pelson** [comment]
This fixed proj.

```
[root@4165174c75ff /]# proj 
proj: /lib64/libc.so.6: version `GLIBC_2.14' not found (required by /opt/conda/bin/../lib/libproj.so.9)
[root@4165174c75ff /]# conda update -c scitools proj4 --force
Fetching package metadata: ......
Solving package specifications: .
Package plan for installation in environment /opt/conda:

The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    proj4-4.9.1                |                1         1.0 MB  scitools

The following packages will be UPDATED:

    proj4: 4.9.1-0 scitools --> 4.9.1-1 scitools

...
[root@4165174c75ff /]# proj
Rel. 4.9.1, 04 March 2015
usage: proj [ -beEfiIlormsStTvVwW [args] ] [ +opts[=arg] ] [ files ]
```

**Comment by pelson** [comment]
Also grib...

```
grib_dump -h
grib_dump: invalid option -- 'h'

NAME 	grib_dump

DESCRIPTION
	Dump the content of a grib file in different formats.

USAGE 
	grib_dump [options] grib_file grib_file ...

OPTIONS
	-O 	Octet mode. WMO documentation style dump.
	-D 	Debug mode.
	-d 	Print all data values.
	-C 	C code mode. A C code program generating the grib message is dumped.
	-t 	Print type information.
	-H 	Print octet content in hexadecimal format.
	-a 	Dump aliases.
	-w key[:{s/d/l}]{=/!=}value,key[:{s/d/l}]{=/!=}value,...	
		Where clause.
		Grib messages are processed only if they match all the key/value constraints.
		A valid constraint is of type key=value or key!=value.
		For each key a string (key:s) or a double (key:d) or a long (key:l)
		type can be specified. Default type is string.
	-M 	Multi-field support off. Turn off support for multiple fields in single grib message.
	-T T | B	Message type. T->GTS, B->BUFR. The input file is interpreted according to the message type.
	-7 	Does not fail when the message has wrong length
	-V 	Version.
```

**Comment by pelson** [comment]
Didn't quite fix iris. Looks like cartopy needs re-building...

**Comment by pelson** [comment]
After #102 we have success!

```
[root@4165174c75ff /]# python -c "import iris; print iris"
<module 'iris' from '/opt/conda/lib/python2.7/site-packages/iris/__init__.pyc'>
```

File: libmo_unpack/build.sh [hunk 1]
```diff
@@ -1,5 +1,8 @@
 #!/bin/bash
 mkdir m4
+
+# Patch the configure.ac as per https://lists.ubuntu.com/archives/fwts-devel/2013-June/003391.html
+sed -i".ac" -e 's/AM_PROG_AR/m4_ifdef([AM_PROG_AR], [AM_PROG_AR])/g' configure.ac
 autoreconf --install
```
Comment (ocefpaf) [inline libmo_unpack/build.sh]: I was trying to download and install  an updated `autoreconf`.  This is much better!

Comment (rhattersley) [inline libmo_unpack/build.sh]: What do you think about SciTools/libmo_unpack#17?

**Pull request merged**